src/v/archival/ntp_archiver_service.cc

```diff
@@ -2171,6 +2206,41 @@ std::ostream& operator<<(std::ostream& os, segment_upload_kind upload_kind) {
     return os;
 }
 
+std::ostream& operator<<(std::ostream& os, flush_response fr) {
+    switch (fr) {
+    case flush_response::accepted:
+        fmt::print(os, "accepted");
+        break;
+    case flush_response::rejected:
+        fmt::print(os, "rejected");
+        break;
+    }
+    return os;
+}
+
+std::ostream& operator<<(std::ostream& os, flush_result fr) {
+    fmt::print(os, "response: {}, offset: {}", fr.response, fr.offset);
+    return os;
+}
+
+std::ostream& operator<<(std::ostream& os, wait_result fr) {
+    switch (fr) {
+    case wait_result::not_in_progress:
+        fmt::print(os, "not in progress");
```

Comment:
there is no formatting here, you can do `return os << "not in progress"`.

Reply:
Fixed.